consensus/service/src/config/tokens.rs

```diff
@@ -11,61 +11,63 @@ use mc_transaction_core::{tokens::Mob, Token, TokenId};
 use std::{fs, iter::FromIterator, path::Path};
 
-mod der_signer_set {
+mod pem_signer_set {
     use super::*;
+    use pem::Pem;
 
-    /// A helper struct for ser/derserializing a SignerSet that uses hex-encoded
-    /// DER representation of Ed25519 public keys.
+    /// A helper struct for ser/derserializing an Ed25519 SignerSet that is PEM
+    /// encoded.
     #[derive(Serialize, Deserialize)]
-    struct DerSignerSet {
-        signers: Vec<String>,
+    struct PemSignerSet {
+        signers: String,
         threshold: u32,
     }
 
-    /// Helper method for serializing a SignerSet<Ed25519Public> into a
-    /// hex-DER-encoded variant.
+    /// Helper method for serializing a SignerSet<Ed25519Public> into PEM.
     pub fn serialize<S: Serializer>(
         signer_set: &Option<SignerSet<Ed25519Public>>,
         serializer: S,
     ) -> Result<S::Ok, S::Error> {
-        let der_signer_set = signer_set.as_ref().map(|signer_set| DerSignerSet {
-            signers: signer_set
+        let pem_signer_set = signer_set.as_ref().map(|signer_set| {
+            let pems = signer_set
                 .signers()
                 .iter()
-                .map(|signer| hex::encode(&signer.to_der()))
-                .collect(),
-            threshold: signer_set.threshold(),
+                .map(|signer| Pem {
+                    tag: String::from("FOO"),
+                    contents: signer.to_der(),
+                })
+                .collect::<Vec<_>>();
+
+            PemSignerSet {
+                signers: pem::encode_many(&pems[..]),
+                threshold: signer_set.threshold(),
+            }
         });
 
-        der_signer_set.serialize(serializer)
+        pem_signer_set.serialize(serializer)
     }
 
-    /// Helper method for deserializing a hex-DER-encoded
-    /// SignerSet<Ed25519Public>.
+    /// Helper method for deserializing a PEM-encoded SignerSet<Ed25519Public>.
     pub fn deserialize<'de, D: Deserializer<'de>>(
         deserializer: D,
     ) -> Result<Option<SignerSet<Ed25519Public>>, D::Error> {
-        let der_signer_set: Option<DerSignerSet> = Deserialize::deserialize(deserializer)?;
-        match der_signer_set {
+        let pem_signer_set: Option<PemSignerSet> = Deserialize::deserialize(deserializer)?;
+        match pem_signer_set {
             None => Ok(None),
-            Some(der_signer_set) => {
-                let signers = der_signer_set
-                    .signers
+            Some(pem_signer_set) => {
+                let pems = pem::parse_many(pem_signer_set.signers.as_bytes())
+                    .map_err(serde::de::Error::custom)?;
+
+                let signers = pems
                     .iter()
-                    .map(|s| {
-                        // Decode each hex-encoded string into a byte vector.
-                        hex::decode(s)
+                    .map(|pem| {
+                        Ed25519Public::try_from_der(&pem.contents[..])
```

Comment:
Would it make sense to check the tag here?

Reply:
I don't think there's a point, the tag doesn't serve any meaningful purpose. Any tag would be fine and will depend on what the user used to encode the PEM.